include/core/utilities.h

```diff
@@ -876,13 +876,7 @@ point_to_angle(const Point<dim> &point,
                    2 * numbers::PI);
 }
 
-/**
- * @brief Converts radius to point in cartesian coordinates (in the x-y plane)
- * @param[in] radius Radial distance
- * @param[in] angle Angle (in radians)
- *
- * @return point Point cartesian coordinates
- */
+
```

Comment:
Why did you remove the documentation of that function? It was well documented. Most likely a mistake.